frontend/packages/console-app/src/components/detect-perspective/useValuesForPerspectiveContext.ts

```diff
@@ -0,0 +1,24 @@
+import * as React from 'react';
+import { isPerspective, Perspective, useExtensions } from '@console/plugin-sdk';
+import { useUserSettingsCompatibility } from '@console/shared';
+import { PerspectiveType } from './perspective-context';
+
+const LAST_PERSPECTIVE_LOCAL_STORAGE_KEY = `bridge/last-perspective`;
+
+const LAST_PERSPECTIVE_USER_SETTINGS_KEY = 'console.lastPerspective';
+
+export const useValuesForPerspectiveContext = (): [
```

Comment:
```suggestion
default export (): [
```
Named files should default export. Generic files `utils...` can export multiple/named.